agent/ebs/watcher.go

```diff
@@ -84,7 +86,6 @@ func (w *EBSWatcher) Start() {
 				w.overrideDeviceName(foundVolumes)
 				if err := w.StageAll(foundVolumes); err != nil {
```

Comment:
I think we can probably remove this check altogether since we'll never return an error in StageAll.